Core/src/main/java/com/paypal/android/core/PayPalJSON.kt

```diff
@@ -0,0 +1,19 @@
+package com.paypal.android.core
+
+import org.json.JSONObject
+
+class PayPalJSON(private val json: JSONObject) {
```

Comment:
This looks good 👍  Maybe a more generic name (PaymentsJSON, CustomJSON, JSONParser?) would be helpful though? I feel like we should reserve the name `PayPal` for classes related to the PayPal payment method, or things will get confusing.

Reply:
Yeah this is a good call i'll rename. I like`PaymentsJSON` to mirror the `BTJSON` concept.